UPGRADING.md

```diff
@@ -0,0 +1,17 @@
+## Upgrading Notes
+This document captures breaking changes between versions of `fastify-jwt`.
+
+### Upgrading from 3.x to 4.0
+
+In `v4` we migrated away from using `jsonwebtoken` to `fast-jwt`. This introduced the following breaking changes:
```

Comment:
If the names of the properties are the only thing remaining.. could we avoid them entirely? We could mske fast-jwt accept both.

Reply:
There is also a breaking change in the `decode` options (`json` entry has been removed and `checkTyp` has been introduced).
This PR will land as a major and apart from those changes and maybe some __exotic__ use cases the migration should be flawless and TBH I highly prefer `fast-jwt` options naming as it is way more obvious.
Accepting both can be confusing and can lead to unexpected behaviours (e.g.: `issuer` and `allowedIss` are both supported in verify options, if by mistake I register something like `{ verify: { issuer: 'one', allowedIss: 'two' } }` which one will be considered valid ?).
In the actual state it's just a matter of searching and replacing a few option names and ts typings will help to spot which ones needs to be updated.

Reply:
I agree with @darkgl0w